app/assets/stylesheets/modules/nlp_parsing.css.scss

```diff
@@ -0,0 +1,39 @@
+// The .tt- prefix is used by the typeahead plugin.  This file
```

Comment:
If we aren't using the typeahead input field anymore, we should use new class names instead of typeahead specific names (i.e "tt-")